libfreerdp/codec/test/TestFreeRDPRegion.c

```diff
@@ -660,6 +663,64 @@ static int test_norbert_case() {
 	return retCode;
 }
 
+static int test_empty_rectangle() {
+	REGION16 region, intersection;
+	int retCode = -1;
+	int i;
+
+	RECTANGLE_16 emptyRectangles[4] = {
+		{   0,    0,    0,    0},
+		{  10,   10,   10,   11},
+		{  10,   10,   11,   10},
+		{ 100,  100,    0,    0}
```

Comment:
OK. I would also fix the 'is_empty' to only check for top=bottom and left=right.